### PR TITLE
chore: test Umi UI e2e test

### PR DESCRIPTION
**Pull request by ycjcl868**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: scripts/test.js [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
-      },
-    );
-    testCmd.on('exit', code => {
-      devServers.forEach(devServer => {
-        if (devServer) {
-          devServer.kill('SIGINT');
-        }
-      });
+    startUiServers()
+      .then(uiServers => {
```
Comment (afc163) [inline scripts/test.js]: then 里面嵌 then

Comment (ycjcl868) [reply scripts/test.js]: fixed

File: scripts/startUiServers.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          child,
+          data: args.data,
+        });
+      }
+    });
+  });
+}
+
+function start() {
+  return Promise.all([startDevServer()]);
```
Comment (afc163) [inline scripts/startUiServers.js]: 这个 Promise.all 貌似没什么必要

Comment (ycjcl868) [reply scripts/startUiServers.js]: 确实

**Pull request merged**